Newsletter 2022 07 (#28)
OBF/newsletter: newsletters/2022-07.md

---
tags: obf
---

_Having trouble viewing this email or looking for a permalink? [View this newsletter in your browser](https://github.com/OBF/newsletter/blob/master/newsletters/2022-07.md)._

_This email was sent to the [bosc-announce](https://groups.google.com/g/bosc-announce) and [open-bio-l](http://mailman.open-bio.org/mailman/listinfo/open-bio-l/) mailing lists._

> # OBF Newsletter July/August 2022

Reviewer: Maybe July/August/September now?

Or just say September and change the filename to match.

Reviewer: ```suggestion
# OBF Newsletter July/August/September 2022
```

Reviewer 2: makes sense

---
tags: obf
---

_Having trouble viewing this email or looking for a permalink? [View this newsletter in your browser](https://github.com/OBF/newsletter/blob/master/newsletters/2022-07.md)._

_This email was sent to the [bosc-announce](https://groups.google.com/g/bosc-announce) and [open-bio-l](http://mailman.open-bio.org/mailman/listinfo/open-bio-l/) mailing lists._

# OBF Newsletter July/August 2022

BOSC 2022 was amazing; BOSC 2023 will be part of ISMB/ECCB in Lyon, France + online. Our next event fellowship application deadline is Oct 1. The Nextflow community is having a get-together, GA4GH wants your thoughts on their Code of Conduct, and educators are invited to use the cloud to teach genomics.

In longer form: 

## OBF News

### BOSC News

> **[BOSC 2022](http://open-bio.org/events/bosc/)** took place in July as a COSI (Community of Special Interest) of [ISMB 2022](https://www.iscb.org/ismb2022), which was held as a hybrid event, in-person in Madison, WI (USA) and online. Over the two days of the BOSC COSI, approximately 200 people participated in [BOSC sessions](https://www.open-bio.org/events/bosc-2022/bosc-2022-schedule/), which included 28 talks chosen from submitted abstracts, 46 posters, and a [panel discussion, “Building and Sustaining Inclusive Open Science Communities”](https://www.open-bio.org/events/bosc-2022/bosc-2022-panel/). BOSC 2022 included [joint keynotes](https://www.open-bio.org/events/bosc-2022/bosc-2022-keynotes/) with two other COSIs. Jason Williams gave a BOSC / Education COSI keynote entitled "Riding the bicycle: Including all scientists on a path to excellence". A joint session with Bio-Ontologies featured a keynote by Melissa Haendel, “The open data highway: turbo-boosting translational traffic with ontologies.”

Author: Great point about the URL, done.

I made sure to spell out COSI in the first line so I don't think we need to spell it out again for the BOSC instance.

> A report about BOSC 2022 is in progress, and will appear as part of the [BOSC collection on F1000](https://f1000research.com/collections/bosc). All BOSC presenters are encouraged to submit their presentation slides to the collection by September 30, 2022.

Reviewer 2: their presentation slides and/or posters

Author: done

> A report about BOSC 2022 is in progress, and will appear as part of the [BOSC collection on F1000](https://f1000research.com/collections/bosc). All BOSC presenters are encouraged to submit their presentation slides and/or posters to the collection by September 30, 2022.

Reviewer 2: ```suggestion
A [report about BOSC 2022](https://f1000research.com/articles/11-1034) has been published as part of the [BOSC collection on F1000](https://f1000research.com/collections/bosc). All BOSC presenters are encouraged to submit their presentation slides and/or posters to the collection by September 30, 2022.
```

Reviewer 2: (I just learned about "suggesting mode" in PR reviews -- super-handy!

You can find personal accounts of BOSC 2022 on the OBF blog: 
- Festus Nyasimi, a co-founder and lead of the Bioinformatics Hub of Kenya Initiative (BHKi), shares the story of [his journey to the ISMB/BOSC 2022 conference on an OBF Event Fellowship](https://www.open-bio.org/2022/08/01/obf-event-fellow2022-fnyasimi/) (see next section for info about Event Fellowships).
- Geraldine Van der Auwera collected impressions and takeaways from multiple BOSC participants in [a blog that will make you wish you were at BOSC 2022 last month and plan to be at BOSC 2023 next year](https://www.open-bio.org/2022/08/16/crowdsourced-highlights-from-bosc-2022/). 

**BOSC 2023** will take place in July as a COSI of [ISMB/ECCB 2023](https://www.iscb.org/ismbeccb2023) as a hybrid event; in-person in Lyon (France), and online.

### OBF Event Fellowships (formerly "Travel Fellowships")

Last round: April 1, 2022  
Next round: October 1, 2022

Our [call for applications is open again, with a **deadline on October 1st**](https://www.open-bio.org/event-awards/). We invite applications for financial support to attend or host events taking place between November 1, 2022 and November 1, 2023.

> 

Reviewer 2: Add link to blog post about call for applications for Oct 1 round? https://www.open-bio.org/2022/08/12/obf-event-fellowship-2022-round2/
(I'm not sure where best to add it)

Author: done, worked it in
